zkevm-circuits/src/evm_circuit/execution/create.rs

```diff
@@ -56,6 +64,9 @@ pub(crate) struct CreateGadget<F> {
 
     initialization_code: MemoryAddressGadget<F>,
     initialization_code_word_size: ConstantDivisionGadget<F, N_BYTES_MEMORY_ADDRESS>,
+    // Init code size must be less than 49152 if Shanghai, otherwise should be
+    // less than 2^40 (maximum value for N_BYTES_MEMORY_ADDRESS bytes).
+    initialization_code_word_size_not_overflow: LtGadget<F, { N_BYTES_MEMORY_ADDRESS + 1 }>,
```

Comment:
```suggestion
    initialization_code_word_size_not_overflow: LtGadget<F, { N_BYTES_MEMORY_ADDRESS }>,
```
If it should be less than 2^40, since `N_BYTES_MEMORY_ADDRESS` is 5, and `LtGadget` does `256**N_BYTES_MEMORY_ADDRESS` so I think this is correct.
Also, see https://github.com/scroll-tech/zkevm-circuits/pull/507/files#diff-66e8458ad6b55447e17de7f715e4395be943d682483be4a945b51f98a4cc50aeR56.